sympy/calculus/util.py

```diff
@@ -1224,7 +1224,8 @@ def __div__(self, other):
                 if S.Zero not in other:
                     return self * AccumBounds(1/other.max, 1/other.min)
 
-                if self.min.is_zero and self.max.is_zero and other.min.is_zero and other.max.is_zero:
+                if (self.min.is_extended_negative and self.max.is_extended_nonnegative or self.min.is_zero or self.max.is_zero) \
```

Comment:
`is_extended_negative or is_zero` can be abbreviated as `is_extended_nonpositive`.